en/aggregate-multiple-cluster-monitor-data.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Aggregate Monitoring Data of Multiple TiDB Clusters
+summary: Learn how to aggregate monitoring data of multiple TiDB clusters by Thanos query.
+---
+
+# Aggregate Monitoring Data of Multiple TiDB Clusters
+
+This document describes how to aggregate the monitoring data of multiple TiDB clusters by Thanos to provide centralized monitoring service.
+
+## Thanos
+
+[Thanos](https://thanos.io/design.md/) is a high availability solution for Prometheus that simplifies the availability guarantee of Prometheus.
+
+Thanos provides [Thanos Query](https://thanos.io/tip/components/query.md/) component as a unified query solution across multiple Prometheus clusters. You can use this feature to aggregate monitoring data of multiple TiDB clusters.
+
+## Configure Thanos Query
+
+First, deploy the TidbMonitor with Thanos Sidecar container:
+
+{{< copyable "shell-regular" >}}
+
+```
+kubectl -n ${namespace} apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/master/examples/monitor-with-thanos/tidb-monitor.yaml"
+```
+
+Then deploy the Thanos Query component:
+
+{{< copyable "shell-regular" >}}
+
+```
+kubectl -n ${namespace} apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/master/examples/monitor-with-thanos/thanos-query.yaml"
+```
+
+In Thanos Query, a Prometheus corresponds to a Store, which corresponds to a TidbMonitor. After deploying Thanos Query, you can provide a unified query interface for monitoring data through Thanos Query's API.
+
+## Configure Grafana
+
+After you deploy Thanos Query, change Grafana's DataSource into Thanos to query the monitoring data of multiple `TidbMonitor` CRs.
+
+## Add or reduce TidbMonitor
+
+If you need to update the `TidbMonitor` CR or take it offline, update the starting configuration `--store` of Thanos Query Store, and perform a rolling update to the Thanos Query component.
```

Comment:
`If you need to update the `TidbMonitor` CR or take it offline`, the original meaning is to add, update, or remove a monitor store from the Thanos query, the configuration is similar with the following:
```
$ thanos query \
    --http-address     "0.0.0.0:9090" \
    --store            "<store-api>:<grpc-port>" \
    --store            "<store-api2>:<grpc-port>" 
```